HuntHelper/Gui/MapUI.cs

```diff
@@ -1411,7 +1411,7 @@ private void UpdateMobInfo()
             _huntManager.AddNearbyMobs(nearbyMobs, _mapZoneMaxCoordSize, _territoryId, MapHelpers.GetMapID(_territoryId),
                 _ttsAEnabled, _ttsBEnabled, _ttsSEnabled, _ttsAMessage, _ttsBMessage, _ttsSMessage,
                 _chatAEnabled, _chatBEnabled, _chatSEnabled, _chatAMessage, _chatBMessage, _chatSMessage,
-                _flyTxtAEnabled, _flyTxtBEnabled, _flyTxtSEnabled, _instance);
```

Comment:
it made me sad to add another parameter to this method, but at this point i just said heck it and did it XD

Reply:
you can use MapHelpers.GetMapName instead of passing thru territoryname (I think they're the same thing?)

Reply:
done!